.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,20 +3,19 @@
 | Branch?       | 4.4 for features / 3.4 or 4.3 for bug fixes <!-- see below -->
 | Bug fix?      | yes/no
 | New feature?  | yes/no <!-- please update src/**/CHANGELOG.md files -->
-| BC breaks?    | no     <!-- see https://symfony.com/bc -->
 | Deprecations? | yes/no <!-- please update UPGRADE-*.md and src/**/CHANGELOG.md files -->
-| Tests pass?   | yes    <!-- please add some, will be required by reviewers -->
-| Fixed tickets | #...   <!-- #-prefixed issue number(s), if any -->
+| Tickets       | Closes #... <!-- #-prefixed issue number(s), if any -->
```

Comment:
> #-prefixed issue number(s), if any

You must use the keyword before each issue you reference for the keyword to work.

maybe something like this?
> prefixed each issue number with the `closes` keyword, if any

Reply:
good idea, updated